src/row.rs

```diff
@@ -30,6 +30,10 @@ impl Symbol {
     pub fn kind(&self) -> SymbolKind {
         self.1
     }
+
+    pub fn id(&self) -> usize {
+        self.0
+    }
```

Comment:
Symbol is an internal type, so you might instead just make it a struct with named fields - this is a non-breaking change. Though it sort of implies that id is mutable. That said, not a big deal. Whatever makes sense on this.